### PR TITLE
added empty lines at EOF and some spaces

### PR DESCRIPTION
**Pull request by ashwin2802**
since i've been requesting this in every PR, it should be done here too

### CODE REVIEW

File: pcl_ransac/include/pcl_ransac/pcl_ransac.hpp [hunk 1]
```diff
@@ -5,25 +5,27 @@
...
 
 class PointCloudModel {
     public:
         PointCloudModel();
         ~PointCloudModel() {};
+        
         virtual void generatePointCloud(pcl::PointCloud<pcl::PointXYZ>::Ptr& cloud) = 0;
-        inline std::vector<int> getModelInliers() const { return inliers_; };
         virtual void fitModel(pcl::PointCloud<pcl::PointXYZ>::Ptr& cloud, const double& distance_threshold) = 0;
+
```
Comment (Kshitij1K) [inline pcl_ransac/include/pcl_ransac/pcl_ransac.hpp:21]: @gajena Are we supposed to leave spaces between functions?

File: pcl_ransac/include/pcl_ransac/pcl_ransac.hpp [hunk 2]
```diff
@@ -5,25 +5,27 @@
 #include <pcl/sample_consensus/ransac.h>
 #include <pcl/sample_consensus/sac_model_plane.h>
 #include <pcl/sample_consensus/sac_model_sphere.h>
-
-#include <Eigen/Core>
-
 #include <pcl_ransac_msgs/ModelCoefficients.h>
+#include <Eigen/Core>
```
Comment (gajena) [inline pcl_ransac/include/pcl_ransac/pcl_ransac.hpp:8]: why it is here, this is the third party

File: pcl_ransac/include/pcl_ransac/pcl_ransac.hpp (hunk 1 above)
Comment (gajena) [reply pcl_ransac/include/pcl_ransac/pcl_ransac.hpp:21]: Depends, usually, I don't leave out a line, but maybe you want to group certain function.

File: pcl_ransac/include/pcl_ransac/pcl_ransac.hpp (hunk 2 above)
Comment (ashwin2802) [reply pcl_ransac/include/pcl_ransac/pcl_ransac.hpp:8]: no oopsie

**Review by gajena — APPROVED** [review]

**Review by Kshitij1K — APPROVED** [review]

**Pull request merged**